--- mussel/mussel	2026-03-13 02:17:02.340155242 +0800
+++ mussel/mussel	2026-03-13 02:18:23.200754752 +0800
@@ -1000,6 +1000,22 @@
   --prefix="" \
   --disable-wrapper >> "$MLOG" 2>&1
 
+printf -- "${BLUEC}..${NORMALC} Adding assembler link if needed ...\n"
+if [ -e $MPREFIX/bin/$XTARGET-ar ]
+then
+  printf -- "${BLUEC}>>${NORMALC} link not needed for AR. \n"
+else
+  printf -- "${BLUEC}>>${NORMALC} link needed for AR. \n"
+  ln -sv $XTARGET-gcc-ar $MPREFIX/bin/$XTARGET-ar >> $MLOG 2>&1
+fi
+if [ -e $MPREFIX/bin/$XTARGET-ranlib ]
+then
+  printf -- "${BLUEC}>>${NORMALC} link not needed for RANLIB. \n"
+else
+  printf -- "${BLUEC}>>${NORMALC} link needed for RAN. \n" 
+  ln -sv $XTARGET-gcc-ranlib $MPREFIX/bin/$XTARGET-ranlib >> $MLOG 2>&1
+fi
+
 printf "${BLUEC}..${NORMALC} Building musl...\n"
 $MAKE \
   AR="$XTARGET-ar" \
